delete.UpdateMailboxAutoReplyConfiguration.ps1

```diff
@@ -78,498 +68,243 @@ function Resolve-HTTPError {
-                    }
-                    else {
-                        [Void]$warningLogs.Add("DryRun: would update mailbox $($aRef.userPrincipalName) ($($aRef.Guid)): $($mailboxSplatParams | ConvertTo-Json)")
-                    }
-                }
-                catch {
-                    $ex = $PSItem
-                    if ( $($ex.Exception.GetType().FullName -eq 'Microsoft.PowerShell.Commands.HttpResponseException') -or $($ex.Exception.GetType().FullName -eq 'System.Net.WebException')) {
-                        $errorObject = Resolve-HTTPError -Error $ex
-                    
-                        $verboseErrorMessage = $errorObject.ErrorMessage
-                    
-                        $auditErrorMessage = $errorObject.ErrorMessage
-                    }
-                    
-                    # If error message empty, fall back on $ex.Exception.Message
-                    if ([String]::IsNullOrEmpty($verboseErrorMessage)) {
-                        $verboseErrorMessage = $ex.Exception.Message
-                    }
-                    if ([String]::IsNullOrEmpty($auditErrorMessage)) {
-                        $auditErrorMessage = $ex.Exception.Message
-                    }
-
-                    [Void]$verboseLogs.Add("Error at Line '$($ex.InvocationInfo.ScriptLineNumber)': $($ex.InvocationInfo.Line). Error: $($verboseErrorMessage)")
-                    $auditLogs.Add([PSCustomObject]@{
-                            Action  = "DeleteAccount"
-                            Message = "Error updating mailbox $($aRef.userPrincipalName) ($($aRef.Guid)): $($mailboxSplatParams | ConvertTo-Json). Error Message: $auditErrorMessage"
-                            IsError = $True
-                        })
-
-                    # Clean up error variables
-                    Remove-Variable 'verboseErrorMessage' -ErrorAction SilentlyContinue
-                    Remove-Variable 'auditErrorMessage' -ErrorAction SilentlyContinue
-                }
-                finally {
-                    $returnobject = @{
-                        mailbox         = $mailbox
-                        aRef            = $aRef
-                        auditLogs       = $auditLogs
-                        verboseLogs     = $verboseLogs
-                        informationLogs = $informationLogs
-                        warningLogs     = $warningLogs
-                    }
-                    $returnobject.Keys | ForEach-Object { Remove-Variable $_ -ErrorAction SilentlyContinue }
-                    Write-Output $returnobject 
-                }
-            }
+    try {
+        Write-Verbose "Updating autoreply configuration for mailbox [$($aRef.userPrincipalName) ($($aRef.Guid))]: $($mailboxSplatParams | ConvertTo-Json)"
+
+        # Get Mailbox "Calendar" folder name
+        $mailboxFolderId = Get-MailboxFolderStatistics -Identity $mailbox.Guid -FolderScope Calendar | Where-Object { $_.FolderType -eq 'Calendar' } | Select-Object Name
```

Comment:
$mailboxFolderId is not used. So I think the Get-MailboxFolderStatistics can be removed

Reply:
Removed